components/ModSubmissionsEditForm.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+    <form @submit.prevent="test" class="p-4">
+        <div>
+            <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                {{ $t('moderateSubmissionForm.contactInformation')}}
+            </span>
+            <div v-for="(field, index) in inputFields.contactFields" :key="index" class="flex flex-col mt-4">
+                <ModInputField  
+                :inputFields="field"
+                />
+            </div>
+        </div>
+
+        <div>
+            <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                {{ $t('moderateSubmissionForm.addresses') }}
+            </span>
+            <div v-for="(field, index) in inputFields.addressesFields" :key="index" class="flex flex-col mt-4">
+                <ModInputField  
+                :inputFields="field"
+                />
+            </div>
+        </div>
+        
+        <div>
+            <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                {{ $t('moderateSubmissionForm.googleMapsInformation') }}
+            </span>
+            <div v-for="(field, index) in inputFields.googleMapsFields" :key="index" class="flex flex-col mt-4">
+                <ModInputField  
+                :inputFields="field"
+                />
+            </div>
+        </div>
+        <button type="submit" class="bg-currentColor text-white font-bold py-2 px-4 rounded">submit</button>
+    </form>
+</template>
+
+<script lang="ts" setup>
+import {ref} from 'vue'
+
+const inputFields = ref({
```

Comment:
Maybe we can store this field object somewhere else, so we can make this mare dynamic, if we need the form for HP we can import the object with all the fields HP need and vica versa.